src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -48,8 +43,32 @@ public void generateReport_CreatesReportFile() {
         assertThat(reportFile).exists();
     }
 
+
+    @Test
+    public void generateReport_DisplaysContentAsEscapedText() {
+        setUpWithJson(HTML_JSON);
+
+        // given
+        page = new FeatureReportPage(reportResult, configuration, features.get(0));
+
+        // when
+        page.generatePage();
+
+        // then
+        DocumentAssertion document = documentFrom(page.getWebPage());
+        assertThat(document.getFeature().getDescription())
+                .isEqualTo("<b>a bold statement</b>");
+        assertThat(document.getFeature().getElements()[0].getDescription())
+                .isEqualTo("<b>a bold statement</b>");
```

Comment:
If you drill down deep enough into `getDescription` you'll see that it uses `TextNode.getWholeText()` which uses the [html text attribute](http://www.w3schools.com/jsref/prop_node_textcontent.asp). This returns the textual representation of the nodes content. Because it is the textual representation any escaped characters are no longer escaped. As it should be, because this is what the user sees.